license.txt

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
Not including the LICENSE would be great :stuck_out_tongue_closed_eyes:

Reply:
yeah I just copied from codepen, so technically you are supposed to carry the license file.. I will remove the readme and the title.

Reply:
Cool works well for us atleast